Cargo.toml

```diff
@@ -68,6 +67,7 @@ version = "0.52.0"
 [target.'cfg(target_os = "macos")'.dependencies]
 cocoa = "0.24.0"
 objc = "0.2.7"
+shlex = "1.1.0"

```

Comment:
nit: appears to be a DOS-style new line here?

Locally I see this.
```
+shlex = "1.1.0"^M
```

Reply:
Weird, because I'm on linux and I don't see it

Reply:
I see it now. The whole file seems to have DOS line endings when I open it with `set ffs=unix` :frowning:, but the diff only shows an added one on that line. I have removed it from that line.